core-common/src/main/java/org/zalando/nakadi/enrichment/MetadataEnrichmentStrategy.java

```diff
@@ -39,12 +40,30 @@ public void enrich(final BatchItem batchItem, final EventType eventType) throws
         }
     }
 
+    @Override
+    public void enrich(final NakadiRecord nakadiRecord, final EventType eventType) throws EnrichmentException {
+        final var metadata = nakadiRecord.getMetadata();
+        metadata.setPublishedBy(getPublisher());
+        final DateTime dateTime = new DateTime(DateTimeZone.UTC);
+        metadata.setReceivedAt(dateTime.toString());
```

Comment:
I thought `received_at` is not a string value, but `long`?

Reply:
changed